R/sf.R

```diff
@@ -622,23 +746,85 @@ sf_rescale01_x <- function(x, range) {
 }
 
 
-#' @param crs Use this to select a specific CRS. If not specified, will
-#'   use the CRS defined in the first layer.
+#' @param crs Use this to select a specific coordinate reference system (CRS).
+#'   If not specified, will use the CRS defined in the first layer.
 #' @param datum CRS that provides datum to use when generating graticules
+#' @param label_axes Character vector or named list of character values
+#'   specifying which graticule lines (meridians or parallels) should be labeled on
+#'   which side of the plot. Meridians are indicated by `"E"` (for East) and
+#'   parallels by `"N"` (for North). Default is `"--EN"`, which specifies
+#'   (clockwise from the top) no labels on the top, none on the right, meridians
+#'   on the bottom, and parallels on the left. Alternatively, this setting could have been
+#'   specified with `list(left = "N", bottom = "E")`.
```

Comment:
Should we have the order be consistent here? (i.e. bottom = "E", left = "N")

Reply:
Will do.